### PR TITLE
how to display plots with CairoMakie

### PR DESCRIPTION
**Pull request by lazarusA**
let's hope this one is also include in the next release... if not. Then version.

### CODE REVIEW

File: contents/data_vis_makie.md [hunk 1]
```diff
@@ -48,3 +48,6 @@ save("filename.pdf", fig; pt_per_unit=0.5)
 
 For `png`'s you specify `px_per_unit`.
 See [Backends & Output](https://makie.juliaplots.org/stable/documentation/backends_and_output/) for details.
+
+Another important issue is to actually visualize your output plot. Note that for `CairoMakie.jl` the Julia REPL is not able to show plots, so you will need and IDE (Integrated Development Environment) such as VSCode, Jupyter or Pluto that supports `png` or `svg` as output.
```
Comment (storopoli) [inline contents/data_vis_makie.md:52]:
```suggestion
Another important issue is to actually visualize your output plot.
Note that for `CairoMakie.jl` the Julia REPL is not able to show plots, so you will need and IDE (Integrated Development Environment) such as VSCode, Jupyter or Pluto that supports `png` or `svg` as output.
```
Consistency :)

Comment (lazarusA) [reply contents/data_vis_makie.md:52]: you mean -so you will need `an` IDE-, could you please update.

**Comment by rikhuijzer** [comment]
Not mentioning an IDE is my biggest regret with the book. It's not only a problem for plots, it's also a problem for working with the REPL and code in files.

Can we add a separate section with pointers to IDEs?

- Easiest way to get started for small projects -> Pluto, Jupyter
- Larger projects -> Visual Studio Code
- Advanced users -> Vim, Emacs

**Comment by lazarusA** [comment]
I agree.

**Pull request merged**